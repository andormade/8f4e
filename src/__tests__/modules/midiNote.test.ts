import moduleTypeGenerator from '../../modules/midiNote';

it('generates correct module type descriptor', () => {
	expect(moduleTypeGenerator()).toMatchSnapshot();
});
