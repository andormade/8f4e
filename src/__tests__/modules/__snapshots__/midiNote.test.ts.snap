// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generates correct module type descriptor 1`] = `
Object {
  "buttons": Array [],
  "category": "Inspection",
  "engine": Object {
    "config": Object {
      "numberOfPorts": 1,
    },
    "name": "buffer",
  },
  "height": 128,
  "initialState": Object {},
  "inputs": Array [
    Object {
      "height": 16,
      "id": "in:1",
      "label": "in",
      "width": 16,
      "x": 0,
      "y": 32,
    },
  ],
  "name": "MIDI Note",
  "outputs": Array [
    Object {
      "height": 16,
      "id": "out:1",
      "label": "out",
      "width": 16,
      "x": 112,
      "y": 32,
    },
  ],
  "sliders": Array [],
  "steppers": Array [],
  "width": 128,
}
`;
